packages/components/src/composite/README.md

```diff
@@ -174,3 +174,48 @@ Allows the component to be rendered as a different HTML element or React compone
 The contents of the component.
 
 -   Required: no
+
+### `Composite.Hover`
+
+Renders an element in a composite widget that receives focus on mouse move and loses focus to the composite base element on mouse leave. This should be combined with the `Composite.Item` component. The `focusOnHover` and `blurOnHoverEnd` props can be used to customize the behavior.
+
+##### `render`: `RenderProp<React.HTMLAttributes<any> & { ref?: React.Ref<any> | undefined; }> | React.ReactElement<any, string | React.JSXElementConstructor<any>>`
+
+Allows the component to be rendered as a different HTML element or React component. The value can be a React element or a function that takes in the original component props and gives back a React element with the props merged.
+
+-   Required: no
+
+##### `children`: `React.ReactNode`
+
+The contents of the component.
+
+-   Required: no
+
+##### `focusOnHover`: `boolean | React.MouseEvent<HTMLElement, MouseEvent>`
+
+Determines if the composite item should be focused when hovered over. Note that the actual DOM focus will stay on the composite element. This item will get the `data-active-item` attribute so it can be styled as if it's focused.
+
+-   Required: no
+-   Default: `true`
+
+##### `blurOnHoverEnd`: `boolean | React.MouseEvent<HTMLElement, MouseEvent>`
+
+Determines if the composite item should lose focus when the mouse leaves. By default, this is set to true if `focusOnHover` is true.
+
+-   Required: no
```

Comment:
I'm not sure if the initial version of `Composite.Hover` should expose the `focusOnHover` and `blurOnHoverEnd`.

I've included them in the docs for now, but we could remove them if we wanted to be more conservative.

Reply:
No strong feelings either way. The Ariakit docs will remain the canonical ones anyway.